projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -65,29 +63,21 @@ def register_notebook_steps(file_path: Path, context: JobContext):
             for jupyter_cell in content["cells"]:
                 if jupyter_cell["cell_type"] == "code":
                     cell = Cell(jupyter_cell)
-                    if CellUtils.is_vdk_cell(cell):
-                        is_sql = CellUtils.is_sql_cell(cell)
-                        cell_type = TYPE_SQL if is_sql else TYPE_PYTHON
-                        runner_func = (
-                            NotebookStepFuncFactory.run_sql_step
-                            if is_sql
-                            else NotebookStepFuncFactory.run_python_step
```

Comment:
nit: so the future implementation could be 
if sql then transform/decorate to execute in python job_input.execute_query(sql)

Reply:
Yes, indeed. Tho the problem is not really with executing the query, but with the fact that VDK works with python not ipython and we get the whole cell content as a string, so we need to recognise all the ipython content by hand and leave only the sql query for vdk which is really error-prone, but ofc it can be done in the future. But it seems like leaving it like that (only with python steps) for now is the best option.